test/tests.json

```diff
@@ -1,689 +1,686 @@
-[
```

Comment:
By changing the indentation you have made it a very difficult task to merge this change.  I have to find a way to diff these two sets and figure out what they are and apply them to the original.

Reply:
Here's the diff
```
diff tests.json.old.normalized tests.json.new.normalized
222a223
>       "header",
440c441
<     "server-disabled": {
---
>     "server": {
445,446c446
<         "--no-restore",
<         "--",
---
>         "--project=Server\/Server.csproj",
456,457c456
<         "--no-restore",
<         "--",
---
>         "--project=Client\/Client.csproj",
461c460
<     "workdir": "netcore\/ThriftTest"
---
>     "workdir": "netcore"
```

Reply:
As usually - some settings of different editors. Maybe you can provide some settings for VSCode in future - this should help to use this standard for Thrift. Can be useful for everyone and for you too.